packages/create-plugin/src/migrations/scripts/001-update-grafana-compose-extend.test.ts

```diff
@@ -0,0 +1,171 @@
+import { describe, it, expect } from 'vitest';
+import { Context } from '../context.js';
+import migrate from './001-update-grafana-compose-extend.js';
+import { parse, stringify } from 'yaml';
+
+describe('001-update-grafana-compose-extend', () => {
```

Comment:
the only thing that came to my mind is that this also preserves comments in the file. could you add a test case for that?